### PR TITLE
WIP Run integration test with Docker

### PR DESCRIPTION
**Pull request by daisy1754**
# Description

Previously we only run integration test with master build because we run test on shared sandbox server and we cannot push random commit. This PR makes our test to run own Docker image - Docker will be setup in CI and we run integration test against actual commit

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [x ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server
tested on CI

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes
- [x] I have created or modified unit tests to sufficiently cover my changes.

### CODE REVIEW

File: tests/scripts/setup_integration.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+echo "Waiting for DB..."
+while ! mysql -uroot -h 127.0.0.1 -e "SELECT 1" >/dev/null 2>&1; do
+    sleep 1
+done
+
+mysql -u root -h 127.0.0.1 circle_test < magento-sample-data-1.9.2.4/magento_sample_data_for_1.9.2.4.sql
+php -d memroy_limit=512M n98-magerun.phar install --magentoVersionByName=magento-mirror-1.9.3.6 \
+  --installationFolder=/var/www/html --forceUseDb --noDownload --dbHost=127.0.0.1 --dbUser=root --dbName=circle_test  --dbPass="" \
+  --installSampleData=yes --useDefaultConfigParams=yes --locale=en_US --baseUrl=http://m1-test.integrations.dev.bolt.me
```
Comment (mccottry) [inline tests/scripts/setup_integration.sh]: @daisy1754 There seems to be a problem with the "--locale=en_US" option in circle CI
https://circleci.com/gh/BoltApp/bolt-magento1/5899?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Comment by adentong** [comment]
@daisy1754 This PR has been open for quite a while now. If this is not being actively worked on, is it ok for me to close it?

**Comment by daisy1754** [comment]
let's close

**Comment by daisy1754** [comment]
please let me keep branch though

**Pull request closed without merging**